client/my-sites/site-settings/press-this-link.jsx

```diff
@@ -84,8 +85,9 @@ class PressThisLink extends React.Component {
 	}
 
 	render() {
+		const omitProps = [ 'site' ];
 		return (
-			<a {...this.props} href={ this.buildPressThisLink() }>
+			<a { ...omit( this.props, omitProps ) } href={ this.buildPressThisLink() }>
```

Comment:
I feel it is more readable this way. Also, this is how it's done at a lot of locations where we omit props, so it feels consistent.